### PR TITLE
Kevin/profile v2 refactor

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description

This PR is to style the relationship tab.
##### Testing
Go to your profile
Click on the relationships tab

### CODE REVIEW

File: app/assets/v2/js/board.js [hunk 1]
```diff
@@ -98,11 +98,34 @@ Vue.mixin({
         vm.disabledBtn = '';
       }
     },
+    removeWorker(key, bountyPk, profileId, obj, section) {
```
Comment (owocki) [inline app/assets/v2/js/board.js]: is this supposed to be on this PR?

**Comment by owocki** [comment]
mind removing the code thats unrelated to the profile refactor? and posting a screenshot?

**Comment by iamonuwa** [comment]
> mind removing the code thats unrelated to the profile refactor? and posting a screenshot?

Sure

File: app/assets/v2/js/board.js (hunk 1 above)
Comment (iamonuwa) [reply app/assets/v2/js/board.js]: I didn't add this

**Comment by iamonuwa** [comment]
image

**Comment by thelostone-mc** [comment]
Any particular reason we have multiple tab variations? thoughts on using an existing one instead ?

**Comment by iamonuwa** [comment]
@owocki do I rework the tabs to reuse existing tabs?

**Comment by owocki** [comment]
lets use the tab designs posted to https://github.com/gitcoinco/web/issues/5119

@iamonuwa do u want to do this in this PR or another PR?

**Comment by iamonuwa** [comment]
> lets use the tab designs posted to #5119
> 
> @iamonuwa do u want to do this in this PR or another PR?

Same PR

**Comment by owocki** [comment]
cool. think you can have a rev by tomo am??

**Comment by iamonuwa** [comment]
Afternoon  😁 atmost

**Comment by owocki** [comment]
how goes?

File: app/assets/v2/js/pages/new_bounty.js [hunk 2]
```diff
@@ -37,11 +37,25 @@ function doShowQuickstart(url) {
...
+    return item.id;
+  })).trigger('change');
+});
+
+$('.select2-clear_invites').on('click', function(e) {
+  e.preventDefault();
+  $('#invite-contributors.js-select2').val(null).trigger('change');
+});
+
+
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/new_bounty.js]: todo: remove extra line

File: app/assets/v2/js/pages/new_bounty.js [hunk 3]
```diff
@@ -37,11 +37,25 @@ function doShowQuickstart(url) {
...
 
 $('.select2-tag__choice').on('click', function() {
   $('#invite-contributors.js-select2').data('select2').dataAdapter.select(processedData[0].children[$(this).data('id')]);
 });
 
+$('.select2-add_byskill').on('click', function(e) {
+  e.preventDefault();
+  $('#invite-contributors.js-select2').val(usersBySkills.map((item) => {
+    return item.id;
+  })).trigger('change');
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/new_bounty.js]: we can refactor this to 
```
$('#invite-contributors.js-select2').val(usersBySkills.map(item => item.id)).trigger('change');
```

File: app/assets/v2/js/board.js [hunk 4]
```diff
@@ -98,11 +98,34 @@ Vue.mixin({
...
+      let vm = this;
+      let url = `/actions/bounty/${bountyPk}/interest/${profileId}/uninterested/`;
+
+      vm.disabledBtn = key;
+      if (window.confirm('Do you want to stop contributor work on this bounty?')) {
+        let postStartpWork = fetchData (url, 'POST');
```
Comment (thelostone-mc) [inline app/assets/v2/js/board.js]: typp ? `postStartWork`

File: app/assets/v2/js/board.js [hunk 5]
```diff
@@ -98,11 +98,34 @@ Vue.mixin({
         vm.disabledBtn = '';
       }
     },
+    removeWorker(key, bountyPk, profileId, obj, section) {
+      let vm = this;
+      let url = `/actions/bounty/${bountyPk}/interest/${profileId}/uninterested/`;
```
Comment (thelostone-mc) [inline app/assets/v2/js/board.js]: `const`

**Pull request closed without merging**